bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/CarouselImpl.java

```diff
@@ -85,6 +85,8 @@
      */
     protected boolean autopauseDisabled;
 
+    private boolean prependControls = false;
```

Comment:
To be in sync with other boolean names, I'd rename it `controlsPrepended`

Reply:
adressed in a407f8573ac5905bd5d9573fdc05fca0a2753c36